backend/src/seeds/creation/index.ts

```diff
@@ -1,23 +1,143 @@
 import { CreationInterface } from './CreationInterface'
 import { nMonthsBefore } from '../factory/creation'
 
-export const creations: CreationInterface[] = [
+const bobsSendings = [
   {
-    email: 'bibi@bloxberg.de',
-    amount: 1000,
+    amount: 10,
     memo: 'Herzlich Willkommen bei Gradido!',
-    creationDate: nMonthsBefore(new Date()),
-    confirmed: true,
-    moveCreationDate: 12,
   },
   {
+    amount: 10,
+    memo: 'von Betty',
```

Comment:
We seed creations, which are not transactions. So all this memos are missleading

Reply:
I don't understand exactly … @Mogge 
What shall I do in your opinion?

Reply:
Thanks for talking! Now I got it …